### PR TITLE
Add cleanup for staticroute

### PR DESCRIPTION
**Pull request by TaoZou1**

### CODE REVIEW

File: pkg/nsx/services/staticroute/staticroute.go [hunk 1]
```diff
@@ -133,3 +133,17 @@ func (service *StaticRouteService) ListStaticRoute() []model.StaticRoutes {
 	}
 	return staticRouteSet
 }
+
+func (service *StaticRouteService) Cleanup() error {
+	staticRouteSet := service.ListStaticRoute()
+	log.Info("cleaning up staticroute", "count", len(staticRouteSet))
+	for _, staticRoute := range staticRouteSet {
+		path := strings.Split(*staticRoute.Path, "/")
```
Comment (dantingl) [inline pkg/nsx/services/staticroute/staticroute.go:141]: Do we need to print every route path when deleting it

Comment (TaoZou1) [reply pkg/nsx/services/staticroute/staticroute.go:142]: done

File: pkg/nsx/services/staticroute/staticroute.go [hunk 2]
```diff
@@ -133,3 +133,19 @@ func (service *StaticRouteService) ListStaticRoute() []model.StaticRoutes {
 	}
 	return staticRouteSet
 }
+
+func (service *StaticRouteService) Cleanup() error {
+	staticRouteSet := service.ListStaticRoute()
+	description := "cleanup staticroute"
```
Comment (seanpang-vmware) [inline pkg/nsx/services/staticroute/staticroute.go:139]: using the same log only print different parameter seems a little strange, could we still print what it is doing in log, eg: "removing static route" in 140

Comment (TaoZou1) [reply pkg/nsx/services/staticroute/staticroute.go]: done

**Review by dantingl — APPROVED** [review]

**Pull request merged**